tests/misc/ShellTest.py

```diff
@@ -91,3 +92,13 @@ def test_run_shell_command_with_stdin(self):
     def test_run_shell_command_kwargs_delegation(self):
         with self.assertRaises(TypeError):
             run_shell_command('super-cool-command', weird_parameter2='abc')
+
+    def test_run_shell_command_with_shell_True(self):
+        run_shell_command('echo "test" > test_shell_True.txt', shell=True)
+        stdout = run_shell_command('cat test_shell_True.txt')[0]
```

Comment:
dont use `cat` for this -- it relies on msys `cat` which not everyone has installed.

`echo` is on all windows by default iirc, but pls make sure this is not executing msys `echo` which iirc acts slightly differently than windows builtin `echo`

Reply:
(if you cant figure out tests without msys, add skip decorators for if these msys binaries do not exist on the test host).